internal/auth/oidc/repository_auth_method_update.go

```diff
@@ -108,12 +104,20 @@ func (r *Repository) UpdateAuthMethod(ctx context.Context, am *AuthMethod, versi
 	opts := getOpts(opt...)
 	if opts.withDryRun {
 		updated := applyUpdate(am, origAm, fieldMaskPaths)
-		err := r.ValidateAuthMethod(ctx, WithAuthMethod(updated))
+		err := r.ValidateDiscoveryInfo(ctx, WithAuthMethod(updated))
 		return updated, db.NoRowsAffected, err
 	}
 
+	// prevent an "active" auth method from being updated in a manner that would create
+	// an incomplete and usable auth method.
```

Comment:
Should "usable" be changed to "unusable" or is there a way for an auth method to be incomplete but still usable?

Reply:
too funny.  that's twice now I've done that.

Reply:
ty!